app/src/main/kotlin/model/io/neo4j/Neo4jRepositoryHandler.kt

```diff
@@ -0,0 +1,84 @@
+package model.io.neo4j
+
+import model.graphs.abstractGraph.Graph
+import org.neo4j.driver.AuthTokens
+import org.neo4j.driver.GraphDatabase
+
+object Neo4jRepositoryHandler {
+    var neo4jRepo: Neo4jRepository? = null
+        private set
```

Comment:
💡 A singleton with a complex state is usually not a good programming pattern since you introduce a global state available from everywhere in the program and it's not clear where it is used